app/src/test/java/org/example/Config.java

```diff
@@ -47,6 +46,8 @@ public static boolean loadConfig(String fileName) throws IOException {
     public static void logConfig() {
         LOGGER.info("udid: " + udid);
         LOGGER.info("appiumUrl: " + appiumUrl);
+        LOGGER.info("userLogin: " + userLogin);
+        LOGGER.info("userPassword: " + userPass);
```

Comment:
```suggestion
        LOGGER.info("userPassword: {}", userPass);
```
Рекомендуемый подход у slf4j через `{}`.

Reply:
> Рекомендуемый подход у slf4j через `{}`.

сделано